src/Form/Type/GiftCardConfigurationType.php

```diff
@@ -37,6 +38,32 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             'allow_delete' => true,
             'by_reference' => false,
         ]);
+        $builder->add('defaultValidityPeriod', DatePeriodType::class, [
+            'label' => 'setono_sylius_gift_card.form.gift_card_configuration.default_validity_period',
+        ]);
+        $builder->get('defaultValidityPeriod')->addModelTransformer(
+            new CallbackTransformer(
+                function (?string $period): array {
+                    $value = null;
+                    $unit = null;
+                    if (null !== $period) {
+                        [$value, $unit] = \explode(' ', $period);
+                    }
+
+                    return [
+                        'value' => $value,
+                        'unit' => $unit,
+                    ];
+                },
+                function (array $data): ?string {
```

Comment:
Shouldn't `array $data` be nullable?

Reply:
No, this is always an array as the child form is composed of 2 inputs. It can not be null